TLM/TLM/UI/Helpers/SliderOption.cs

```diff
@@ -64,7 +64,7 @@ public SliderOption(string fieldName, Options.PersistTo scope = Options.PersistT
 
         public override void Load(byte data) => Value = data;
 
-        public override byte Save() => (byte)Mathf.Clamp(Value, 0, 255);
+        public override byte Save() => (byte)Mathf.RoundToInt(Value);
```

Comment:
This entirely changes the semantic of the code.
Clamp was checking the byte bounds, while RoundToInt does not and it may (and will) fall out of byte range.
While you have moved Clamp to the setter, at least have a debug assertion here.

Reply:
Just updated with some improvements, including the `Save()` method.